CHANGELOG.md

```diff
@@ -4,8 +4,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
+This project uses [*towncrier*](https://towncrier.readthedocs.io/) and
+the changes for the upcoming release can be found in
+<https://github.com/twisted/my-project/tree/main/changelog.d/>.
```

Comment:
Is this URL configurable somewhere and did it get missed?

Reply:
Yep! Fixed